ohapi/public.py

```diff
@@ -77,6 +77,7 @@ def download_url(result, directory, max_bytes):
                 f.write(chunk)
 
     logging.info('Downloaded {}'.format(filename))
+    return
```

Comment:
I'm unclear on why this bare return is needed or wanted here?

Reply:
Actually, the files upon being downloaded never exited the function. So I was just trying to test the function by putting a return but it seems that multiple copies of the same files is download (I tested only for 23andme though) which is another issue. Yeah this return statement is not required here. I will remove it. :)